password_enhancements.module

```diff
@@ -0,0 +1,471 @@
+
+    $type = $constraint->getType();
+
+    /** @var \Drupal\password_enhancements\Plugin\PasswordConstraintPluginInterface $plugin_instance */
+    try {
+      $plugin_instance = $plugin_manager->createInstance($type, $constraint->getConfiguration());
+      $initial_description = $plugin_instance->getInitialDescription();
+    }
+    catch (Exception $e) {
+      Drupal::service('logger.password_enhancements')->exception($e);
+
+      $initial_description = strtr('Failed to create <em>:type</em> plugin instance.', [
+        ':type' => $type,
+      ]);
+
+      // Create an error constraint to show the that this has issues.
+      $id = 'error';
+      $type = 'error';
+      $constraint = new Constraint([
+        'type' => $type,
+        'required' => TRUE,
+        'descriptionSingular' => $initial_description,
+      ], 'password_enhancements_constraints');
+    }
+
+    // Get JS library if set.
+    $plugin_definition = $plugin_manager->getDefinition($type);
+    if (!empty($plugin_definition['jsLibrary'])) {
+      $libraries[] = $plugin_definition['jsLibrary'];
+    }
+    $plugin_config[$type][$id] = $constraint->getConfiguration();
+
+    $arguments = [];
+    foreach ($constraint->getSettings() as $setting => $value) {
+      $arguments["@{$setting}"] = new FormattableMarkup('<span data-setting="@setting">@value</span>', [
+        '@setting' => $setting,
+        '@value' => $value,
+      ]);
+    }
+
+    $processed_constraints[] = [
+      '#theme' => 'password_enhancements_policy_constraint',
+      '#description' => strtr($initial_description, $arguments),
+      '#attributes' => [
+        'id' => $id,
+        'class' => ['constraint'],
+        'data-required' => $constraint->isRequired() ? 'yes' : 'no',
+        'data-validation-passed' => 'no',
+        'data-constraint' => $constraint->getType(),
+      ],
+    ];
+    unset($constraints[$constraint->id()]);
+  }
+
+  // Set wrapper for the constraints.
+  $ajax_wrapper = 'password-policy-constraint-ajax-wrapper';
+  $form['account']['password_policy_constraints'] = [
+    '#weight' => $form['account']['pass']['#weight'] + 0.001,
+    '#prefix' => "<div id='{$ajax_wrapper}'>",
+    '#suffix' => '</div>',
+  ];
+
+  // Try to load policy by its role and priority.
+  try {
+    /** @var \Drupal\password_enhancements\Entity\Storage\PolicyEntityStorageInterface $password_policy_storage */
+    $password_policy_storage = Drupal::entityTypeManager()->getStorage('password_enhancements_policy');
+    $policy = $password_policy_storage->loadByRoleAndPriority($roles);
+  }
+  catch (Exception $e) {
+    Drupal::service('logger.password_enhancements')->exception($e);
+    $policy = NULL;
+    $form['account']['password_policy_constraints'] += [
+      '#type' => 'container',
+      '#markup' => t('Failed to load policy.'),
+      '#attributes' => [
+        'id' => 'logged-out-message',
+        'class' => ['messages', 'messages--error'],
```

Comment:
Should not this be rendered as a normal Drupal message? If it should, you should use the `#theme => 'status_message'` theme instead. https://github.com/Pronovix/swagger_ui_formatter/blob/efe1e212ccfcecd95f08e508cd09c31fcbeafbc5/src/Plugin/Field/FieldFormatter/SwaggerUIFormatterTrait.php#L161-L162

Reply:
Or should it be themed as `password_enhancements_error_messages`?

Reply:
Hm, something got mixed up here, I'll fix.

Reply:
Fixed.